src/azure-cli/azure/cli/command_modules/botservice/_params.py

```diff
@@ -60,10 +60,9 @@ def load_arguments(self, _):
         c.argument('description', options_list=['--description', '-d'], help='The description of the bot.', arg_group='Registration Bot Specific')
         c.argument('endpoint', options_list=['-e', '--endpoint'], help='The messaging endpoint of the bot.', arg_group='Registration Bot Specific')
         c.argument('msa_app_id', options_list=['--appid'], help='The Microsoft account ID (MSA ID) to be used with the bot.')
-        c.argument('password', options_list=['-p', '--password'], help='The Microsoft account (MSA) password for the bot. Used to authorize messages being sent to the bot.')
+        c.argument('password', options_list=['-p', '--password'], help='The Microsoft account (MSA) password for the bot. Used to authorize messages being sent to the bot. Required for Web App Bots.', arg_group='Web App Bot Specific')
```

Comment:
What do you want to show for your help message with "arg_group='Web App Bot Specific"? Could you make it more clear?